examples/otlp/docker/docker-compose.yaml

```diff
@@ -0,0 +1,58 @@
+version: "2"
+services:
+
+  # Jaeger
+  jaeger-all-in-one:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - "16686:16686"
+      - "14268"
+      - "14250"
+
+  # Zipkin
+  zipkin-all-in-one:
+    image: openzipkin/zipkin:latest
+    ports:
+      - "9411:9411"
+
+  # Collector
+  otel-collector:
+    image: ${OTELCOL_IMG}
+    command: ["--config=/etc/otel-collector-config-demo.yaml", "${OTELCOL_ARGS}"]
+    volumes:
+      - ./otel-collector-config-demo.yaml:/etc/otel-collector-config-demo.yaml
+    ports:
+      - "1888:1888"   # pprof extension
+      - "8888:8888"   # Prometheus metrics exposed by the collector
+      - "8889:8889"   # Prometheus exporter metrics
+      - "13133:13133" # health_check extension
+      - "55678"       # OpenCensus receiver
+      - "55681:55679" # zpages extension
+      - "55680:55680" # otlp receiver
+    depends_on:
+      - jaeger-all-in-one
+      - zipkin-all-in-one
+
+  # Agent
+  otel-agent:
```

Comment:
It may be confusing for new users, what is Collector and what is Agent and why do they need both. Some documentation may be of help.

Reply:
Hm. I hear what you're saying, but this example isn't supposed to be documentation about the collector. It's just supposed to be a plug-and-play way to show things working. Not even the collector demo itself has this documentation (although I think that one definitely should!)

Reply:
removed the agent to make it less confusing.